website/docs/guides/container_service_extension_4_x_cluster_management.html.markdown

```diff
@@ -36,47 +36,49 @@ section to understand how it works.
+* `auto_repair_on_errors`: Setting this to `true` will make the CSE Server to automatically repair the TKGm cluster on errors.
 
 The following four placeholders are **only needed if one wants to provide a default storage class** with the TKGm cluster.
-If this is not needed, please remove the whole `defaultStorageClassOptions` block from the JSON template:
+If this is not needed, please remove the whole `defaultStorageClassOptions` block from [the JSON template][tkgmcluster_template]:
 
-- `default_storage_class_filesystem`: Filesystem for the default storage class. Only `ext4` or `xfs` are valid.
-- `default_storage_class_name`: Name of the default storage class, it must contain only lowercase alphanumeric characters or '-',
+* `default_storage_class_filesystem`: Filesystem for the default storage class. Only `ext4` or `xfs` are valid.
+* `default_storage_class_name`: Name of the default storage class, it must contain only lowercase alphanumeric characters or '-',
   start with an alphabetic character, end with an alphanumeric, and contain at most 63 characters.
-- `default_storage_class_storage_profile`: Storage profile to use for the default storage class, for example `*`.
-- `default_storage_class_delete_reclaim_policy`: Set this to `true` to use a "Delete" reclaim policy, that deletes the volume when the PersistentVolumeClaim is deleted.
+* `default_storage_class_storage_profile`: Storage profile to use for the default storage class, for example `*`.
+* `default_storage_class_delete_reclaim_policy`: Set this to `true` to use a "Delete" reclaim policy, that deletes the volume when the PersistentVolumeClaim is deleted.
 
 To create a valid input for the `capi_yaml` placeholder, a [CAPVCD][capvcd] YAML is required, which describes the TKGm cluster to be
 created. In order to craft it, we need to follow these steps:
 
-- First, we need to download a YAML template from the [CAPVCD repository][capvcd_templates].
-  We should choose the template that matches the TKGm OVA. For example, if we uploaded the `ubuntu-2004-kube-v1.22.9+vmware.1-tkg.1-2182cbabee08edf480ee9bc5866d6933.ova`
-  and we want to use it, the template that we need to obtain corresponds to v1.22.9, that is `cluster-template-v1.22.9.yaml`.
+* First, we need to download a YAML template from the [CAPVCD repository][capvcd_templates].
+  We should choose the template that matches the TKGm OVA. For example, if we uploaded the `ubuntu-2004-kube-v1.25.7+vmware.2-tkg.1-8a74b9f12e488c54605b3537acb683bc.ova`
+  and we want to use it, the template that we need to obtain corresponds to v1.25.7, that is `cluster-template-v1.25.7.yaml`.
 
-- This template requires some extra elements to be added to the `kind: Cluster` block, inside `metadata`. These elements are `labels` and
+* This template requires some extra elements to be added to the `kind: Cluster` block, inside `metadata`. These elements are `labels` and
```

Comment:
Please help understand why the downloaded YAML doesn't have all the missing pieces below to begin with? Why do we have to add these snippets? Is there a way to save the user from this?

Reply:
That is a good question. I have some guesses, like the Certificate parts (because CSE v4.1 allows having custom certificates), but I'll try to get a better answer from CSE team.

Reply:
In summary, is that the CAPI templates need to leverage a lot of use cases, being these (the missing parts) some specifics that CSE use.